.circleci/config.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+
+jobs:
```

Comment:
Can you maybe add a comment in this file, that this is for doc building and that the upload takes ~30min for future reference?

Reply:
Done

Reply:
Thanks!